src/transformers/pipelines/base.py

```diff
@@ -77,7 +77,7 @@ def _pad(items, key, padding_value, padding_side):
         # Others include `attention_mask` etc...
         shape = items[0][key].shape
         dim = len(shape)
-        if key == "pixel_values":
+        if key in ["pixel_values", "image"]:
```

Comment:
`DocumentQuestionAnsweringPipelineTests.get_test_pipeline` prepare inputs with "`image` key.

- not sure why not `pixel_values` there.
-  `DocumentQuestionAnsweringPipelineTests` failed after #20426, because of the usage of `LayoutLMv2`, The tests with this model were all skipped previously (i.e. before #20426)